plugin/coala.vim

```diff
@@ -1,40 +1,40 @@
 " g:linters_automatic_on_save is a global flag for enabling/disabling the
```

Comment:
we use spaces everywhere in coala and my vim is configured to only use spaces :P project policy? It's easier if people don't have to switch for this project